rules/java_rules.build_defs

```diff
@@ -249,14 +248,9 @@ def java_binary(name:str, main_class:str=None, out:str=None, srcs:list=None, dep
             toolchain = toolchain,
         )
         deps += [lib_rule]
-    # TODO(jpoole): Remove self exec flag and do this always once JavaBinaryExecutableByDefault has been merged
-    if self_executable:
-        preamble = '#!/bin/sh\nexec java %s -jar $0 ${@}' % (jvm_args or '')
-        cmd, tools = _jarcat_cmd(main_class, preamble, manifest=manifest)
-    else:
-        # This is essentially a hack to get past some Java things (notably Jersey) failing
-        # in subtle ways when the jar has a preamble (srsly...).
-        cmd, tools = _jarcat_cmd(main_class, manifest=manifest)
+    # This is essentially a hack to get past some Java things (notably Jersey) failing
+    # in subtle ways when the jar has a preamble (srsly...).
+    cmd, tools = _jarcat_cmd(main_class, manifest=manifest)
```

Comment:
Should this not be the other way around? This is the non-executable version of it